irohad/consensus/yac/impl/yac.cpp

```diff
@@ -85,11 +85,21 @@ namespace iroha {
                                  [](const auto &p) { return p->address(); }),
                        ", "));
 
+        std::vector<std::vector<VoteMessage>> states;
         std::unique_lock<std::mutex> lock(mutex_);
         cluster_order_ = order;
         alternative_order_ = std::move(alternative_order);
         round_ = hash.vote_round;
+        auto it = future_states_.lower_bound(round_);
```

Comment:
Could you tell what does this block of code do in hunan lang?

Reply:
```suggestion
        // take states from current round if it was stored, erase older than current.
        auto it = future_states_.lower_bound(round_);
```
May be it is worth to create a named function or lambda for future readers generations.